src/Visualization/Vtk.jl

```diff
@@ -94,10 +94,12 @@ function create_vtk_file(
 
   if num_cells(trian)>0
     for (k,v) in celldata
-      vtk_cell_data(vtkfile, _prepare_data(v), k)
+      comp_names = _data_component_names(v)
```

Comment:
I would replace this by
```julia
component_names = _data_component_names(v)
vtk_cell_data(vtkfile, _prepare_data(v), k; component_names)
```
and idem everywhere else.

Reply:
Oh I didn't know this trick, I changed these.

Reply:
Actually the setindex overload is so cursed that I cannot avoid repeating the kwarg name in it :') .